site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,233 @@
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div id="global-header-1.1" class="navbar-collapse collapse me-lg-auto global-header-1">
+        <ul class="navbar-nav">
+          <li class="nav-item"><a class="nav-link" href="#">All inclusive cards</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Data only cards</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Top up</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Assistance</a></li>
+        </ul>
+        <ul class="navbar-nav border-top border-1 border-dark d-flex flex-row d-lg-none">
+          <li class="nav-item"><a class="nav-link active" href="#" aria-label="EN English version" aria-current="true">EN</a></li>
+          <li class="nav-item"><a class="nav-link" href="#" aria-label="FR Version française" lang="fr" hreflang="fr">FR</a></li>
+        </ul>
+      </div>
+      <div id="global-header-1.2" class="navbar-collapse collapse d-sm-flex global-header-1">
+        <ul class="navbar-nav flex-row">
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"></use>
+              </svg>
+              <span class="visually-hidden">Basket</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#user"></use>
+              </svg>
+              <span class="visually-hidden">My account</span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+</header>
+<main class="bd-content order-1">
+  <div class="bg-white title-bar">
+    <div class="container-xxl d-block">
+      <div class="row">
+        <div class="col-sm-12 col-lg-5 offset-lg-3">
+          <h1 class="display-1">Become a reseller</h1>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-xxl pt-4">
+    <div class="row">
+      <div class="col-sm-12 col-lg-5 offset-lg-3">
+        <p class="ll-sm h3 mb-4">You are a professional and wish to resell Orange products to your
+          customers? Please fill the form and we will get back to you shortly.
+        </p>
+        <form class="needs-validation mb-5" novalidate>
+          <div class="row">
+            <div class="col-12">
+              <p class="fw-bold mb-4">
+                <span class="text-primary">*</span> indicates required
```

Comment:
Isn't it better to use `.ms-1` here instead of a white space? I'm thinking about the RTL version we'll need to create. BTW let's create an issue to develop the RTL version.
```suggestion
                <span class="text-primary ms-1">*</span>indicates required
```

Reply:
`me-1` seems better to have a whitespace after the '*'?

Reply:
Yep my bad of course we got to use `.me-1` 😇